test/test_sparse_csr.py

```diff
@@ -1498,6 +1499,51 @@ def run_test_block_addmm_addmv(self,
         self.assertEqual(actual, out)
         self.assertEqual(actual, expected)
 
+    @parametrize("block_size", [16, 32, 64])
+    @parametrize("index_dtype", [torch.int32, torch.int64])
+    @unittest.skipIf(not has_triton(), "Triton is not available")
+    @onlyCUDA
+    @dtypes(torch.half, torch.bfloat16)
+    @dtypesIfCUDA(*[torch.half] if SM53OrLater else [],
+                  *[torch.bfloat16] if SM80OrLater else [])
+    def test_triton_bsr_dense_bmm(self, device, dtype, index_dtype, block_size):
+        from functools import partial
+        from torch.sparse.triton_ops import bsr_dense_mm
+
+        tensor = partial(make_tensor, device=device, dtype=dtype, low=0.5, high=1.5)
+        # Tensor params
+        batches = [(), (2,)]
+        size = [128, 256]
+
+        kernel_run = [False]
+
+        def mm_kernel(*args, **kwargs):
+            kernel_run[0] = True
+            return bsr_dense_mm(*args, **kwargs)
+
+        # Kernel overwrite
+        lib = torch.library.Library("aten", "IMPL")
+        lib.impl("aten::_triton_bsr_dense_mm", mm_kernel, "SparseCsrCUDA")
```

Comment:
wait, so we are registering the kernel in the test only, and it won't be available otherwise?

Reply:
cc @anjali411, @SherlockNoMad for registering on import

Reply:
@anjali411 , @SherlockNoMad, could you please tell me where it is best to register a kernel overwrite?

Reply:
Hi yeah so I think you wanna add it in `torch/__init__.py` similar to how we add _meta_registrations at import time here: https://github.com/pytorch/pytorch/blob/5faa2792fa3c46f2124d1d1c5f7b6a3865d47d7b/torch/__init__.py#L1136